scripts/check_version

```diff
@@ -36,13 +56,36 @@ if [[ x${STRIPPED_PACKAGE_VERSION}x != x${IOS_MARKETING_VERSION_2}x ]]; then
 	EXITCODE=-1
 fi
 
-# This will check the format of IOS' CURRENT_PROJECT_VERSION
-if [[ ! "$IOS_PROJECT_VERSION_1" =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
-    echo "Invalid format (expected X.X.X) for first CURRENT_PROJECT_VERSION (${IOS_PROJECT_VERSION_1}) on ios/HathorMobile.xcodeproj/project.pbxproj"
+# Check the format of iOS' CURRENT_PROJECT_VERSION
```

Comment:
The patch number of the `CURRENT_PROJECT_VERSION` may be different from 0. If there's a build error, we can increase this patch number and try to build it again, without changing the release/candidate version.

This should be reflected in the checks, we must accept `x1.0.{anynumber}x`, and after `x0.${RCVERSION}.{anynumber}x`

Reply:
Fixed on [7ae829a](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/406/commits/7ae829acd3f9dfcfb9ef80aee50f34408d81cc28)